src/internal/components/chart-status-container/index.tsx

```diff
@@ -54,26 +55,29 @@ export default function ChartStatusContainer({
   isEmpty,
   showChart,
 }: ChartStatusContainerProps) {
+  const i18n = useInternalI18n('[charts]');
+
   const statusContainer = useMemo(() => {
     const handleRecoveryClick = (event: CustomEvent) => {
       event.preventDefault();
       fireNonCancelableEvent(onRecoveryClick);
     };
     if (statusType === 'error') {
+      const renderedRecoveryText = i18n('recoveryText', recoveryText);
       return (
         <span>
-          <InternalStatusIndicator type="error">{errorText}</InternalStatusIndicator>{' '}
-          {recoveryText && (
+          <InternalStatusIndicator type="error">{i18n('errorText', errorText)}</InternalStatusIndicator>{' '}
+          {renderedRecoveryText && (
```

Comment:
Interesting: are we removing the possibility to hide the recovery text? Are there cases where there shouldn't be text here?